### PR TITLE
PXB-1962: Release notes - v8.0.8

### PR DESCRIPTION
**Pull request by borys-belinsky**
PXB-1915: Missing options - add

### CODE REVIEW

File: storage/innobase/xtrabackup/doc/.gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 build/
 source/ext/psdom.pyc
 source/percona-theme/
+COMMIT_MESSAGE
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/.gitignore:4]: why this is needed?

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+================================================================================
+
+- An encrypted table could not be restored when ``ADD INDEX`` or ``DROP INDEX``
+  commands had been run on the table. Bug fixed :jirabug:`PXB-1905`
+- In some cases ``xtrabackup --prepare`` could fail to decrypt a table but
+  reported that the operation completed ok. Bug fixed :jirabug:`PXB-1936`
+- ``xtrabackup --move-back`` did not complete successfully when the encrypted
+  binlog file. Bug fixed :jirabug:`PXB-1937`.
+- |pxb.name| could crash during the prepare stage when making an incremental
+  backup when a multi valued index was being added added or dropped for JSON
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst]: added added

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  data. Bug fixed :jirabug:`PXB-1913`.
+- A backup could fail when the database was on a fast storage and the redo
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst]: no need RN. it was a bug in new feature which we release in this version

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+================================================================================
+
+- Support log archiving feature in PXB 8.0. More information in
+  :jirabug:`PXB-1912` and in the `Redo Log section of MySQL documentation
+  <https://dev.mysql.com/doc/refman/8.0/en/innodb-redo-log.html>`_
+- Two options (`–-backup-lock-timeout` and `-–backup-lock-retry-count`) were
+  added to enable the configuring of the timeout for acquiring metadata locks in
+  ``FLUSH TABLES WITH READ LOCK``, ``LOCK TABLE FOR BACKUP``, and ``LOCK BINLOG
+  FOR BACKUP`` statements. More information in :jirabug:`PXB-1914`
+
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst:35]: we should mention https://jira.percona.com/browse/PXB-1915 as new feature

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst [hunk 5]
```diff
@@ -720,6 +720,15 @@ Options
...
 
+.. option:: --rocksdb-checkpoint-max-age
+
+   The checkpoint cannot be older than this number of seconds when the backup
+   completes.
+
+.. option:: --rocksdb-checkpoint-max-count
+
+   Complete the backup even if the checkpoint age requirement has not been met after
+   this number of checkpoints.
```
Comment (gl-sergei) [inline storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst:740]: `–-backup-lock-timeout` and `-–backup-lock-retry-count` should be mentioned

**Review by gl-sergei — CHANGES_REQUESTED** [review]

File: storage/innobase/xtrabackup/doc/.gitignore (hunk 1 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/.gitignore:4]: This is a file to collect the information about the changes that belong to the commit. It is handy to have the file around while updating the documentation. The file itself does not belong to documentation hence it is in .gitignore

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst (hunk 5 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst]: Need more information about these options

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst (hunk 4 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst:35]: Added

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst (hunk 2 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst]: Fixed

File: storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst (hunk 3 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.8.rst]: Removed

File: storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst (hunk 5 above)
Comment (borys-belinsky) [reply storage/innobase/xtrabackup/doc/source/xtrabackup_bin/xbk_option_reference.rst:740]: Added based on PXB-1914

**Review by gl-sergei — APPROVED** [review]

**Pull request merged**